frontend/src/pages/NewRun.test.tsx

```diff
@@ -38,19 +37,23 @@ class TestNewRun extends NewRun {
 }
 
 describe('NewRun', () => {
+  const consoleErrorSpy = jest.spyOn(console, 'error');
   const createJobSpy = jest.spyOn(Apis.jobServiceApi, 'createJob');
   const createRunSpy = jest.spyOn(Apis.runServiceApi, 'createRun');
   const getExperimentSpy = jest.spyOn(Apis.experimentServiceApi, 'getExperiment');
   const getPipelineSpy = jest.spyOn(Apis.pipelineServiceApi, 'getPipeline');
   const getRunSpy = jest.spyOn(Apis.runServiceApi, 'getRun');
   const historyPushSpy = jest.fn();
   const historyReplaceSpy = jest.fn();
-  const listPipelinesSpy = jest.spyOn(Apis.pipelineServiceApi, 'listPipelines');
```

Comment:
How come we don't need this? Shouldn't we get an error that `fetch` has failed when the pipeline selector opens? Or do we never use mount in tests below, never actually rendering the selector?

Reply:
Not sure. There are some tests where we fully mount the tree, and interact with the selector, but it doesn't seem to be a problem. perhaps it's related to the peculiarities of `Dialog` that was preventing us from testing the "dismiss" paths?